### DISCUSSION TITLE
Unable to install brew into .linuxbrew folder inside a users folder

### DISCUSSION THREAD

**Discussion by bdivakaran** [Homebrew/discussions#4016] [Linux] [answered]
I started using homebrew about 2 years ago on my enterprise linux servers.
At my work, they provide us with RHEL server for working but they don't contain any of the latest tools.
We don't have sudo permissions either on these servers to install anything on our own as everything is maintained by IT.
 
Homebrew has been a godsend for me as I could install it into /users/$USER/.linuxbrew and get a lot of awesome new CLI tools.
When I used to install brew it would give me an option of either installing to /home/linuxbrew with sudo or /users/$USER/.linuxbrew without sudo which is what i used to choose by pressing ctrl+D

But now when I try to install brew on a new server, I don't see an option of installing to /users/$USER/.linuxbrew anymore and only see the option to install it to /users/linuxbrew and with sudo only.
I do see brew asking me to use alternate intall methods but they don't seem to be working well for me and I keep running into some issues.

Can someone help in this regard. I don't seem to find any clear reason as to why it was stopped and if the option will comeback or do i need to enable a flag/environment variable.

**Comment by bdivakaran** [comment]
Can someone please help me in this regard?

**Comment by SMillerDev** [comment, marked as answer]
It was stopped because usage was never tested so it was broken about as often as it worked which means treating it as an alternate install was the better way.

**Reply by bdivakaran** [reply]
Thank you for the info.
I'll experiment with alternative installs
